plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/model/Plugin.java

```diff
@@ -476,7 +478,22 @@ public Path getLogFile() {
      * @return Marker object
      */
     public Marker getMarker() {
-        return MarkerFactory.getMarker(name);
+        Path logsPath;
```

Comment:
Why we don't use getLogFile() from Plugin ? I would expect to see it updated and return the new location